samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/SamLite_eu.properties

```diff
@@ -133,3 +133,15 @@ samlite_feedbackOK=Ongi
 samlite_feedbackNOK=Ez zuzena
 samlite_true=Egiazkoa
 samlite_false=Faltsua
+
+#S2U-22
```

Comment:
@pablosanroman You might want to review these texts, thanks!

Reply:
Hello @mpellicer , reviewed here! thanks!


rich_text_label=Erakutsi testu aberastuaren editorea (eta karaktere zenbatzailea)

rich_editor_instructions_label=Testu aberastuaren editorearen argibideak
rich_editor_instructions_prelude=Editore honek aukera ematen du galderak sortzerakoan html etiketa desberdinak onartzeko. Modu honetan, testuak sortu daitezke letra lodiz edo etzanez, azpimarratu daiteke, irudiak gehitu eta abar. 
rich_editor_instructions_prelude_2=Baina editore hau erabiltzeko hainbat xehetasun hartu behar dituzu kontuan:
rich_editor_instructions_recomendation_1=Estilo gehigarria erabiltzea pentsatuta baduzu, gomendagarria da hasieratik erabiltzea, horrela, aldaketa egitean, idatzitako guztia lerro bakar batean konbinatuko baita. 
rich_editor_instructions_recomendation_2=Estilo gehigarriak aplikatuko dira soilik galdera tituluetan, iruzkinetan edo hainbat galderen erantzunetan; gainerakoetan ez dute inolako eraginik izango.
rich_editor_instructions_recomendation_3=Galdera definitzeko ez da erabili behar [Galdera 12 (1 puntu) (0.5 deskontua)]. Galdera hori sortzean akatsak eragin ditzakeelako.
rich_editor_instructions_recomendation_4=Iruzkinetarako erabiltzen denean ('\#FBOK' eta '\#FBNOK') '\:' puntuazio-markaren ondoren bakarrik erabili behar da, aurretik erabiltzen bada ez da ongi aplikatuko
rich_editor_instructions_recomendation_5=Erantzunetarako erabiltzen denean ('a. Erantzuna' edo '*b. Erantzun zuzena') puntuazio markatik aurrera bakarrik erabili behar da '.', aurretik erabiltzen bada ez da ongi aplikatuko

Reply:
Thank you!